galleries/examples/lines_bars_and_markers/linestyles.py

```diff
@@ -8,7 +8,9 @@
 ``(offset, (on_off_seq))``. For example, ``(0, (3, 10, 1, 15))`` means
 (3pt line, 10pt space, 1pt line, 15pt space) with no offset, while
 ``(5, (10, 3))``, means (10pt line, 3pt space), but skip the first 5pt line.
-See also `.Line2D.set_linestyle`.
+See also `.Line2D.set_linestyle`.  The specific on off sequence of the
+"dotted", "dashed" and "dashdot" styles may be configured using
+``rcParams["lines.{style}_pattern"]``.
```

Comment:
The f-string replacer here is a little mysterious - maybe add an example?
```suggestion
``rcParams["lines.{style}_pattern"]``, eg. `rcParams["lines.dotted_pattern"] = (0, (1, 10))`. 
```

Reply:
What do you think about listing them all out with the `:rc:` directive?  I quite like that we can see the defaults for free, but not sure if it's just too much...

Reply:
sure, explicit is probably fine.